apps/formbricks-com/app/docs/contributing/gitpod/page.mdx

```diff
@@ -1,15 +1,97 @@
 import Image from "next/image";
+import GitpodPorts from "./gitpod-ports.webp";
+import GitpodAuth from "./gitpod-auth.webp";
+import GitpodNewWorkspace from "./gitpod-new-workspace.webp";
+import GitpodPreparing from "./gitpod-preparing.webp";
+import GitpodRunning from "./gitpod-running.webp";
 
 export const meta = {
   title: "Gitpod Setup",
   description:
     "With one click, you can setup the Formbricks developer environment in your browser using Gitpod",
 };
 
+#### Contributing
+
+# Gitpod
+
 ### One Click Setup
 
 1. Click the button below to open this project in Gitpod.
 
 2. This will open a fully configured workspace in your browser with all the necessary dependencies already installed.
 
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/formbricks/formbricks)
```

Comment:
Can we shift this between the point 1 and 2 to make it true to the "click the button below"?

Reply:
sure